src/map/ai/controllers/mob_controller.cpp

```diff
@@ -955,6 +955,7 @@ bool CMobController::MobSkill(uint16 targid, uint16 wsid)
     if (POwner)
     {
         FaceTarget(targid);
+        PMob->PAI->EventHandler.triggerListener("WEAPONSKILL_BEFORE_USE", PMob, wsid);
```

Comment:
wondering if we should leave it weaponskill or say mobskill instead to further ifferentiate between player ws.

Reply:
Do we use "MOBSKILL" in the other listener handlers? If so, that's my vote

Reply:
alas, t'would appear we do NOT so changing just this one would result in inconsistency.